docs/content/guide/getting-started.md

```diff
@@ -152,25 +152,33 @@ docker exec -it mongo-1 mongo admin --eval 'rs.initiate({
 - Setup spica
 
 ```sh
-# Start spica (client) on port 8080
-docker run --name spica --network spica -p 8080:80 -d gcr.io/spica-239113/spica
-# Start spica (api) on port 4300
+# Start spica (API) on port 4400
 docker run
- --name api
- --network spica
- -p 4300:4300
- -e PORT=4300
- -e DATABASE_URI="mongodb://mongo-1,mongo-2,mongo-3"
- -e REPLICA_SET="rs0"
- -e DATABASE_NAME=spica
- -e PUBLIC_HOST=http://localhost:4300 # You can replace this with your domain name if you have any.
- -e PERSISTENT_PATH=/tmp # The path that the storage files will be kept at.
- -d gcr.io/spica-239113/api
-# Important: you need to apply these two commands until
-# https://github.com/spica-engine/spica/issues/20 resolves.
-docker exec -it spica sed -i "s/\/spica\//\//g" /usr/share/nginx/html/index.html
-# Basically, this replaces /api with http://localhost:4300 which is our accesible api url.
-docker exec -w /usr/share/nginx/html -it spica find . -type f -iname 'main-es*.js' -exec sed -i 's/\"\/api\"/\"http:\/\/localhost:4300\"/g' {} \;
+    --name api
+    --network spica
+    -p 4400:80
+    -e PORT=80
+    -e DATABASE_URI="mongodb://mongo-1,mongo-2,mongo-3"
+    -e REPLICA_SET="rs0"
+    -e DATABASE_NAME=spica
+    # Password of the default spica user
+    -e DEFAULT_PASSWORD=spica
+    # API will use this secret to sign login tokens.
+    -e SECRET="$2b$10$shOzfYpDCy.RMgsVlwdQeONKGGzaBTfTQAjmXQNpMp4aKlLXrfZ/C"
+    # This secret should is important for security of your spica.
+    # You should change this secret into something else in production.
```

Comment:
"# This secret is important to keep your Spica secure."
"# Changing this secret to something different is advised."